homework.4/homework.4.playground/Contents.swift

```diff
@@ -0,0 +1,55 @@
+// homework.4
+
+import UIKit
+
+var tupleArray = [(100, "Continue"), (202, "Accepted"), (203, "Non-Authoritative Information"),
+                  (302, "Found"), (307, "Temporary Redirect"), (404, "Not Found")]
+
+// Task 1
+
+func squaringIntegerElement(_ : [(Int, String)]) -> [Int] {
+    var newTupleArray: [Int] = []
+    
+    for index in 0..<tupleArray.count {
+        newTupleArray.append(Int(pow(Double(tupleArray[index].0), 2)))
+    }
+    
+    return newTupleArray
+}
+
+// Task 2
+
+func writingEvenElements(_ : [(Int, String)]) -> [(Int, String)] {
+    var newTupleArray: [(Int, String)] = []
+
+    for index in 0..<tupleArray.count {
```

Comment:
We need to go through elements with cycle in this task not through elements indices